### DISCUSSION TITLE
installed on wordpress already but what is the next step?

### DISCUSSION THREAD

**Discussion by harismateen** [Codeinwp/logo-maker#78] [Q&A]
Hi, I have installed the plugin directly into the wordpress, but how should i put this into my page?

**Comment by Soare-Robert-Daniel** [comment]
Very simple. Make a completely empty page in Gutenberg Editor, and the Logo Maker will be available as a Block. You insert it and preview the page to see if it looks good. If it does not, you might have other elements on the page that you still have to remove.

**Reply by Cri00** [reply]
Hi,

Any possibility to add it in a Wordpress page with shortcode?

Thanks!

**Reply by Cri00** [reply]
Got it:

```
<!-- wp:themeisle/logo-maker -->
<div id="themeisle-logo-maker-root" class="wp-block-themeisle-logo-maker"></div>
<!-- /wp:themeisle/logo-maker -->
```
